pkg/cmd/server/server.go

```diff
@@ -295,9 +245,18 @@ func NewCmdServer() *cobra.Command {
 			if err != nil {
 				return err
 			}
-			err = o.run(cmd)
-			cobra.CheckErr(err)
-			return nil
+
+			cancel := util.InitCmd(cmd, &logutil.Config{
+				File:                 o.serverConfig.LogFile,
+				Level:                o.serverConfig.LogLevel,
+				FileMaxSize:          o.serverConfig.Log.File.MaxSize,
+				FileMaxDays:          o.serverConfig.Log.File.MaxDays,
+				FileMaxBackups:       o.serverConfig.Log.File.MaxBackups,
+				ZapInternalErrOutput: o.serverConfig.Log.InternalErrOutput,
+			})
+			defer cancel()
+
+			return runServer(o)
```

Comment:
@crelax Can we use `return runServer(o)` directly and mark the `SilenceUsage` of the command as false to avoid printing the help message (https://github.com/pingcap/tiflow/pull/5524).

Reply:
Ref: https://github.com/pingcap/tiflow/pull/5686/commits/0ab05f46be5cf475abd62764afdbc28fe55fb6cf

Reply:
So sorry that I just noticed this pr and replied late. 
Yes, it can avoid printing redundant help messages by marking `SilenceUsage`. Nevertheless, when the user's inputs are lacking or invalid, silence mode also eliminates the usage tips and that's the time we consider the tips necessary. :)

Reply:
We may want to throw an error and print usage tips when the command failed for invalid inputs, and remain silent and just report the error when the error occurs within `run` function since irrelevant to users' inputs.

Reply:
@crelax Got it, I will revert the `cobra.CheckErr` part.

Reply:
Address in commit https://github.com/pingcap/tiflow/pull/5686/commits/cd66f542d4507c71597f8b52c4435bb3a3cbd340